Tasks/VsTest/vstest.ts

```diff
@@ -51,23 +52,22 @@ try {
     if (useNewCollectorFlag && useNewCollectorFlag.toUpperCase() == "TRUE") {
         useNewCollector = true;
     }
-    
+
     var releaseuri = tl.getVariable("release.releaseUri")
-	var context = "CI";
-	if(releaseuri)
-	{
-		context = "CD";
-	}
+    var context = "CI";
+    if (releaseuri) {
+        context = "CD";
+    }
 
     var systemDefaultWorkingDirectory = tl.getVariable('System.DefaultWorkingDirectory');
     var artifactsDirectory = tl.getVariable('System.ArtifactsDirectory');
     var testAssemblyFiles = getTestAssemblies();
```

Comment:
Shouldn't we be able to use classes in typescript ?

---
In reply to: [87553898](https://github.com/Microsoft/vsts-tasks/pull/3065#discussion_r87553898) [](ancestors = 87553898)